test/acceptance/helpers/helpers.go

```diff
@@ -0,0 +1,54 @@
+package helpers
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/hashicorp/consul/sdk/testutil/retry"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func RandomName() string {
```

Comment:
To some of these, I didn't add docs because they seemed "self-explanatory". I'm curious what you think. Do you think we should generally add comments/docs to all functions? For example, `KubectlDelete` function just does what its name is, so the doc for it would seem like it will just rephrase the function name as an English sentence. Perhaps, I'm missing something though.

Reply:
Well randomname seems self explanatory but I see it prepends `test-`. I think it'll be easier to have a blanket "document all exported helpers" convention than have each eng decide for themselves.